viewing_party/party.py

```diff
@@ -1,23 +1,193 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    #if any parameter is empty return none
+    if title and genre and rating:
+        new_movie = {"title": title, "genre": genre, "rating": rating}
+        return new_movie
+    else:
+        return None

```

Comment:
Nice work in creating the movie dictionary based on `truthy` values! 

Another way to write this function is by using a guard clause and building the dictionary directly in the return statement. This allows your function to exit early when `falsy` values are passed in and reduces the amount of nested code we have, allowing the primary purpose of the function to be slightly easier to read.
 ```python
def create_movie(title, genre, rating):
    if not title or not genre or not rating:
        return None
        
    return { 
            "title": title, 
            "genre": genre, 
            "rating": rating 
        }
```